core/src/err.rs

```diff
@@ -42,6 +57,11 @@ impl Error {
                     _ => 0,
                 }
             }
+            Error::NoDatabase
+            | Error::InvalidResultSetJsonSchema,
```

Comment:
```suggestion
            | Error::InvalidResultSetJsonSchema
```
Extra unneeded comma here

Reply:
good catch!